docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
If the channel were "likes", it would get flooded with likes for every entity on the site. How to have pages without lots of channels (e.g. activity page) is a tough problem.